packages/ui/package.json

```diff
@@ -18,7 +18,7 @@
     },
     "dependencies": {
         "@fluentui/react": "^8.96.1",
```

Comment:
Please update @fluentui/react version to ^8.118.1. I think this got missed for ui package during react 18 changes

Reply:
We will raise separate PR for this change.